gradle/localdb.gradle

```diff
@@ -0,0 +1,3 @@
+task localdb(type: Exec) {
+  commandLine "sh", "-c", "docker run -d --rm --name cash-mysql-57 -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=true mysql:5.7"
```

Comment:
If you use 3306 this will fail for people running mysql locally already, I would pick a new port if you're going to do this

Reply:
Also I don't think this will be typical development flow for most people, I think most will have their own version running already. I don't think this needs to be packed as something into misk

Reply:
Good point, I'll update it so a port can be passed in.